resources/assets/components/InboxItem/index.js

```diff
@@ -52,12 +70,12 @@ class InboxItem extends React.Component {
           </ul>
         </div>
         <div className="container__block -third">
-          {post['user'] ?
+          {post['user'] || signup.user ?
             <div>
-              <h2>{post['user']['first_name']} {post['user']['last_name']}, {calculateAge(post['user']['birthdate'])}</h2>
+              <h2>{first_name} {last_name}, {birthdate}</h2>
```

Comment:
when I tried to do the above for the Campaign Inbox page (`showInbox` function) and logged out the user, I got 
```
Object
  birthdate: "1989-04-12"
  displayName: "5978afc27f43c27d8755558c"
```
which seemed to grab the user ID instead of display name?